src/pages/experiments/[experimentId]/plots-and-tables/embedding-categorical/index.jsx

```diff
@@ -31,8 +31,8 @@ const EmbeddingCategoricalPage = ({ experimentId }) => {
   const hierarchy = useSelector(getCellSetsHierarchy());
 
   useEffect(() => {
+    if (!config) dispatch(loadPlotConfig(experimentId, plotUuid, plotType));
     dispatch(loadCellSets(experimentId));
```

Comment:
Add checks for the cellsets too?

Reply:
`loadCellSets` has an internal check which would not fire the request if cell set exists: https://github.com/hms-dbmi-cellenics/ui/blob/master/src/redux/actions/cellSets/loadCellSets.js#L13-L15 So we don't have to explicitly guard this